test/jdk/java/awt/datatransfer/Independence/IndependenceAWTTest.java

```diff
@@ -65,46 +88,44 @@ public void focusGained(FocusEvent fe) {
         tf2 = new TextField();
         tf3 = new TextField();
 
-        panel = new Panel();
+        Panel panel = new Panel();
         panel.setLayout(new BorderLayout());
 
         panel.add(tf2, BorderLayout.NORTH);
         panel.add(tf3, BorderLayout.SOUTH);
 
         frame.add(tf1, BorderLayout.NORTH);
         frame.add(panel, BorderLayout.CENTER);
+        frame.setLocationRelativeTo(null);
 
         frame.setVisible(true);
         tf1.requestFocus();
     }
 
     // Get System Selection i.e. Primary Clipboard
-    private void getPrimaryClipboard() {
-        Properties ps = System.getProperties();
-        String operSys = ps.getProperty("os.name");
+    private static void getPrimaryClipboard() {
         try {
-            pClip = Toolkit.getDefaultToolkit().getSystemSelection();
-            if (pClip == null)
-                if ((operSys.substring(0,3)).equalsIgnoreCase("Win") || operSys.toLowerCase().contains("os x"))
-                    System.out.println(operSys + "Operating system does not support system selection ");
-                else
-                    throw new RuntimeException("Method getSystemSelection() is returning null on X11 platform");
-        } catch(HeadlessException e) {
+            primaryClip = Toolkit.getDefaultToolkit().getSystemSelection();
+            if (primaryClip == null) {
+                throw new RuntimeException("Method getSystemSelection() is returning null"
+                                           + " on Linux platform");
+            }
+        } catch (HeadlessException e) {
             System.out.println("Headless exception thrown " + e);
         }
     }
 
     // Method to get the contents of both of the clipboards
-    public void getClipboardsContent() throws Exception {
-        sClip = Toolkit.getDefaultToolkit().getSystemClipboard();
+    private static void getClipboardsContent() throws Exception {
+        systemClip = Toolkit.getDefaultToolkit().getSystemClipboard();
         Transferable tp;
         Transferable ts;
 
         StringSelection content = new StringSelection(tf1.getText());
-        sClip.setContents(content,content);
+        systemClip.setContents(content, content);
 
-        tp = pClip.getContents(this);
-        ts = sClip.getContents(this);
+        tp = primaryClip.getContents(null);
+        ts = systemClip.getContents(null);
```

Comment:
Since the requestor param is no longer used in Clipboard's [getContents()](https://github.com/openjdk/jdk/blob/7c944ee6f4dda4f1626721d63ac6bc6d1b40d33b/src/java.datatransfer/share/classes/java/awt/datatransfer/Clipboard.java#L146), it has been changed to null.
primaryClip.getContents(null)